packages/cli/src/index.js

```diff
@@ -97,6 +106,78 @@ config({
   multiline: true,
 })
 
+const TELEMETRY_ENABLED =
+  telemetry !== 'false' && !process.env.REDWOOD_DISABLE_TELEMETRY
```

Comment:
Is that how yargs actually parses it here, as a string?

Reply:
If I recall correctly I did have to do some debugging around this when I wrote this in create redwood-app. (We use this same code in there too btw). This ended up working for my local testing/tinkering. Can double check for you though.

Reply:
I see what's going on—we don't tell yargs how to parse this at all here:

https://github.com/redwoodjs/redwood/blob/b3644c3cb2b889678352773a7d5ad298d5531e43/packages/cli/src/index.js#L68

so yeah what you've done is great